dbt_project.yml

```diff
@@ -0,0 +1,36 @@
+name: 'snowplow_media_player'
+version: '0.1.0'
+config-version: 2
+require-dbt-version: ">=1.0.0"
+
+profile: 'default'
+
+model-paths: ["models"]
+analysis-paths: ["analyses"]
+test-paths: ["tests"]
+seed-paths: ["seeds"]
+macro-paths: ["macros"]
+docs-paths: ["docs"]
+snapshot-paths: ["snapshots"]
+
+target-path: "target"
+clean-targets:
+  - "target"
+  - "dbt_packages"
+
+vars:
+  snowplow__percent_progress_boundaries: [10, 25, 50, 75]
+  snowplow__valid_play_sec: 30
+  snowplow__complete_play_rate: 0.99
+  snowplow__max_media_pv_window: 2
+
+models:
+  snowplow_media_player:
+    +bind: false
+    +materialized: table
+    web:
+      +tags: "snowplow_web_incremental"
+    custom:
```

Comment:
Would you want to disable `custom` by default? Seems like the logical default value but up to you

Reply:
good point, did not consider this yet but it is the logical thing to do